db_router.py

```diff
@@ -0,0 +1,34 @@
+
+
+class TmpBaseRouter(object):
+    """
+    A router to control all database operations on models in the
+    auth application.
+    """
+    def db_for_read(self, model, **hints):
+        """
+        Attempts to read auth models go to auth_db.
+        """
+        if model._meta.model_name == 'search':
```

Comment:
What about letting Model register them-selves for this router? I'm thinking about adding a `Meta` property like `router = "xxxx"`? Or maybe `read_router/write_router` but that'd be a bit overkill for now.